### PR TITLE
Change apartment page header

### PR DESCRIPTION
**Pull request by thuypham03**
This PR means to change the apartment page header (issue #153) by:
- Adding address of the apartment (if different from the apartment's name) below its name
- Move number of reviews and heart rating section to under the apartment's photo
- Relocation reviews and heart rating section form Apartment/Header.tsx to ApartmentPage.tsx

Problem need to resolve in other PR:
- Relocate the sort and leave a review so that heart rating and number of review can line up in a line
Screen Shot 2022-03-31 at 3 34 36 PM

### CODE REVIEW

**Comment by thuypham03** [comment]
issue #153

**Review by samliu10 — CHANGES_REQUESTED** [review]
Looks good except make sure the reviews and hearts are on the same line. See designs
Screen Shot 2022-03-31 at 3 45 57 PM

**Comment by thuypham03** [comment]
@samliu10  for this commit, I try to put the review and heart rate on the same line, but I cannot find way to also put the buttons on the same line as well, can you help me take a look of it, the code of this part is in frontend/src/pages/ApartmentPage.tsx (line 197-273)

**Comment by samliu10** [comment]
I actually think this is a question for @shelost because there is not enough room for everything to be on one line— could you make a design for what this should look like at different breakpoints? One idea is to have the review information + "leave a review" button be on the same line at opposite ends, and then move the review sorting to the second line.

File: frontend/src/pages/ApartmentPage.tsx [hunk 1]
```diff
@@ -184,13 +203,30 @@ const ApartmentPage = (): ReactElement => {
 
   const Header = (
     <>
-      <Grid container item spacing={3} justify="space-between" alignItems="center">
-        <Grid item>
-          <Typography variant="h4">Reviews ({reviewData.length})</Typography>
-          {reviewData.length === 0 && (
-            <Typography>No reviews available. Be the first to leave one!</Typography>
+      <Grid container alignItems="center">
```
Comment (samliu10) [inline frontend/src/pages/ApartmentPage.tsx:206]: Based on new designs, I changed it to look like this
Screen Shot 2022-04-21 at 8 55 58 PM

Here's a video showing resizing: 

https://user-images.githubusercontent.com/57203639/164580926-953cf0cf-6844-47e7-bced-1c91dec9a2ad.mov

File: frontend/src/components/Review/Review.tsx [hunk 2]
```diff
@@ -69,51 +69,44 @@ const ReviewComponent = ({
...
-                        aria-expanded={expanded}
-                        aria-label="show more"
-                        size="small"
-                      >
-                        <ExpandMoreIcon />
-                      </IconButton>
-                    </Grid>
-                  </Grid>
-                ) : (
-                  <Grid item>
```
Comment (samliu10) [inline frontend/src/components/Review/Review.tsx:98]: Removed because there are no reviews from Ithaca Tenant Union

**Review by samliu10 — APPROVED** [review]

**Pull request merged**